flake.nix

```diff
@@ -13,6 +13,7 @@
       apple-macbook-pro-11-5 = import ./apple/macbook-pro/11-5;
       apple-macbook-pro-12-1 = import ./apple/macbook-pro/12-1;
       apple-macbook-pro-14-1 = import ./apple/macbook-pro/14-1;
+      apple-macmini-4-1 = import ./apple/macmini/4;
```

Comment:
Please also add the model to the README.md.

Reply:
Done.